### ISSUE TITLE
Feature request: eCRFs that can be submitted multiple times

### ISSUE THREAD

**Issue by josharmenta** [phoenixctms/ctsms#331]
It would be nice if we could mark certain eCRFs that can be submitted in a proband's file multiple times per trial, such as an Adverse Event report or similar forms which might be required during the course of a clinical trial.

**Comment by rkrenn** [comment]
yes, well known request - extensive change/not trivial to implement.
we will try to raise funds, and schedule it for 2025.

**Comment by josharmenta** [comment]
Sounds good. Let me know how I can help.

I had a thought while looking through the code that it might be simpler to implement it as a separate submodule of the trial with separate tables, maybe called "Trial Forms" or something like that. Then it wouldn't require a large rewrite of the eCRF module.
